packages/mgt-element/src/providers/Providers.ts

```diff
@@ -112,6 +116,55 @@ export class Providers {
     return this._me;
   }
 
+  /**
+   * Gets the cache ID, creates one if it does not exist
+   *
+   * @static
+   * @memberof Providers
+   */
+  public static async getCacheId(force?: boolean) {
```

Comment:
If you clear the cacheId before firing the activeaccountchanged event, then you don't need a forced attribute